scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -12,42 +12,21 @@ workflow GvsCreateFilterSet {
     String call_set_identifier
 
     String filter_set_name
-    Array[String] indel_recalibration_annotation_values = ["AS_FS", "AS_ReadPosRankSum", "AS_MQRankSum", "AS_QD", "AS_SOR"]
-    Array[String] snp_recalibration_annotation_values = ["AS_QD", "AS_MQRankSum", "AS_ReadPosRankSum", "AS_FS", "AS_MQ", "AS_SOR"]
 
     File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
     File? gatk_override
 
     Boolean use_classic_VQSR = true
+
+    Array[String] indel_recalibration_annotation_values = ["AS_FS", "AS_ReadPosRankSum", "AS_MQRankSum", "AS_QD", "AS_SOR"]
+    Array[String] snp_recalibration_annotation_values = ["AS_QD", "AS_MQRankSum", "AS_ReadPosRankSum", "AS_FS", "AS_MQ", "AS_SOR"]
```

Comment:
If any of these inputs are used for VQSR classic / lite only can they be named or commented accordingly for the sanity of future maintainers?

Reply:
I've updated naming.